src/components/header/header.tsx

```diff
@@ -88,26 +123,33 @@ const LinkLogo = styled(Link)`
 const NavigationMenu = styled.div`
   display: flex;
   align-items: center;
+  margin-left: 1rem;
 `
 const NavigationMenuItem = styled.div`
-  a {
+  > * {
+    padding-left: 0.5rem;
+    padding-right: 0.5rem;
+  }
+  > a {
+    display: flex;
+    align-items: center;
+    height: 60px;
     color: var(--textSecondaryColor);
-    font-size: 1rem;
+    font-size: 16px;
```

Comment:
Moving away from using `rem`?

Reply:
Thank you, I've removed the hard-coded size, we prefer using `rem` indeed,

In a very close future I think I will set the "base" font-size to `16px` instead of `14px`, it makes calculations easier (`14px` being `0.875rem` if the base is `16px`)